### DISCUSSION TITLE
Hacktoberfest 2023

### DISCUSSION THREAD

**Discussion by NyorJa** [trebol-ecommerce/trebol-backend-monolith#257] [General]
@bglamadrid what do you think?

**Comment by bglamadrid** [comment]
Hi @NyorJa hey been a long time! Hope it was a nice year for you. And yep, it's that time of the year again! I'll try to housekeep things and prepare some good first issues, but it'll take me some time.

As always, contributions are more than welcome. Check the frontend too if you're interested.

**Comment by NyorJa** [comment]
I will put our repo also in this https://github.com/OSSPhilippines/hacktoberfest-2023-participants so that I can invite other devs to my country. Anyway just take your time in housekeeping and lets roll!

**Comment by bglamadrid** [comment]
As it turns out, during October I moved out to start living on my own, and just couldn't take care of this. 
Still, I hope that you had a good time and made some to contribute elsewhere, Nyorja.
And as always, thanks for your heartwarming interest in open source, and this project specifically :)
